exasol/python_extension_common/deployment/language_container_builder.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import annotations
+from typing import Callable
+import shutil
+import subprocess
+import tempfile
+from pathlib import Path
+from importlib import resources
+
+from exasol_integration_test_docker_environment.lib.docker.images.image_info import ImageInfo   # type: ignore
+from exasol_script_languages_container_tool.lib import api            # type: ignore
+from exasol_script_languages_container_tool.lib.tasks.export.export_containers import ExportContainerResult     # type: ignore
+
+
+def exclude_cuda(line: str) -> bool:
+    return not line.startswith("nvidia")
+
+
+def find_path_backwards(target_path: str | Path, start_path: str | Path) -> Path:
+    """
+    An utility searching for a specified path backwards. It begins with the given start
+    path and checks if the target path is among its siblings. Then it moves to the parent
+    path and so on, until it reaches the root of the file structure. Raises a FileNotFound
+    error if the search is unsuccessful.
+    """
+    current_path = Path(start_path).parent
+    while current_path != current_path.root:
+        result_path = Path(current_path, target_path)
+        if result_path.exists():
+            return result_path
+        current_path = current_path.parent
+    raise FileNotFoundError(f"Could not find {target_path} when searching backwards from {start_path}")
+
+
+def copy_slc_flavor(dest_dir: str | Path) -> None:
+    """
+    Copies the content of the language_container directory to the specified
+    destination directory.
+    """
+    files = resources.files(__package__).joinpath('language_container')
+    with resources.as_file(files) as pkg_dir:
+        shutil.copytree(pkg_dir, dest_dir, dirs_exist_ok=True)
```

Comment:
This is not guaranteed to work, depending on how the package is installed. We should have code in the ITDE which extracts resource directories properly.

Reply:
It is supposed to be reliable. For example, it should still work if the package is in a zip. The `resources.as_files` extract the content of the resource and copy it into a temporary directory where I can access it using the normal Path.

Reply:
Ah, misinterpreted it.